app/views/staff/implementing_organisations/new.html.haml

```diff
@@ -0,0 +1,19 @@
+= content_for :page_title_prefix, t("page_title.activity.implementing_organisation.new", name: @activity.title)
+
+= link_to t("generic.link.back"), organisation_activity_path(@activity.organisation, @activity), class: "govuk-back-link"
+%main.govuk-main-wrapper#main-content{ role: "main" }
+  .govuk-grid-row
+    .govuk-grid-column-two-thirds
+      %h1.govuk-heading-xl
+        = t("page_title.activity.implementing_organisation.new")
+
+      = form_with model: @implementing_organisation, url: activity_implementing_organisations_path(@activity) do |f|
+        = f.govuk_error_summary
+        = f.hidden_field :activity_id, value: @activity.id
```

Comment:
I got confused for a second here as `participating-org` also has a field called `activity-id` that is used to allow us to point to a related activity down the tree! http://reference.iatistandard.org/202/activity-standard/iati-activities/iati-activity/participating-org/#example-usage

I don't think we can avoid this for the moment unless we rename `activity` to be more explicit in `parent_activity`. We aren't putting this value into the XML which was the main worry I had 👍

Reply:
Yeah - I regret not calling it 'parent_activity' - we should definately do that refactor, we have too much 'activity' already!